packages-common/pluggable-widgets-tools/tests/commands.js

```diff
@@ -0,0 +1,175 @@
+const { Mutex } = require("async-mutex");
+const { exec } = require("child_process");
+const fkill = require("fkill");
+const { copy, readJson, writeJson } = require("fs-extra");
+const { join } = require("path");
+const { mkdir, rm, tempdir } = require("shelljs");
+const { promisify } = require("util");
+const { run: runYeoman } = require("yeoman-test");
+
+const LIMIT_TESTS = process.argv.includes("--limited");
+
+const CONFIGS = [
```

Comment:
Is there reasoning behind why these configs are picked?

Reply:
It is combination of risk (8.0 has a higher chance of failure), importance (8.6 and latest are the one people stick to the most), and a desire to have less configs to make tests faster. At the end it is a gut-feeling based cherry-pick of combinations, so that "we cover recent ts/js/native/web", "everything in 8.6", "js in versions close to 8.6 that covers only ts", etc...